### ISSUE TITLE
Put together some kind of press package

### ISSUE THREAD

**Issue by fossecode** [BustByte/coronastatus.no#559]
With images and info about the project. We should try get some publicity in order to gain traction.

**Comment by jelbazi** [comment]
A friend owns a presskit service. Maybe he can offer it for free for Coronastatus. I'll ask if he's interested.

**Comment by fossecode** [comment]
Great!

**Comment by baskoopmans** [comment]
Yes for sure @jelbazi and @fossecode, please signup on https://presskite.com/ and you will be upgraded to a premium account without charge 👍

**Comment by fossecode** [comment]
> Yes for sure @jelbazi and @fossecode, please signup on https://presskite.com/ and you will be upgraded to a premium account without charge

Awesome, thanks! Registered with eirik@bustbyte.no

**Comment by baskoopmans** [comment]
Perfect, both your accounts are upgraded 🎉
